project1/server/server_main.cpp

```diff
@@ -13,22 +13,27 @@
  */
 void PrintUsageAndExit(const char* program_name) {
```

Comment:
(minor) It would be nice to initialize logging here according to [this page](https://github.com/emilk/loguru)...